### ISSUE TITLE
[Problem] 10001st prime number

### ISSUE THREAD

**Issue by ademclk** [ademclk/callofcode#32]
### 10001st prime number 

**First 5 prime numbers are: 2, 3, 5, 7, 11**
**5th prime is: 11.**

> Find the 10001st prime number.

**Comment by Legoeggolas** [comment]
Looks fun, I'll do it for C++

**Comment by knshkp** [comment]
am want to work on that

**Comment by ademclk** [comment]
No problem. Waiting for your solution.

**Comment by jacobjohnso** [comment]
Hello, I'm also interested in doing this task. Can you assign me this issue?

**Comment by ademclk** [comment]
Done.

**Comment by s11ngh** [comment]
> > Hi, I can work on this. Could you assign me this issue?
> 
> Done.

Thanks! Can I submit the solution in Python or Java? I see there's already a C++ solution

**Comment by ademclk** [comment]
> > > Hi, I can work on this. Could you assign me this issue?
> > 
> > 
> > Done.
> 
> Thanks! Can I submit the solution in Python or Java? I see there's already a C++ solution

Yes you can submit any language you want. If you've different approach you can submit the cpp version, too.

**Comment by s11ngh** [comment]
> > > > Hi, I can work on this. Could you assign me this issue?
> > > 
> > > 
> > > Done.
> > 
> > 
> > Thanks! Can I submit the solution in Python or Java? I see there's already a C++ solution
> 
> Yes you can submit any language you want. If you've different approach you can submit the cpp version, too.

Great, I just submitted a python solution with a proposed (more) efficient solution in the description. I would love to implement that later as well, but I can't do that due to time constraints right now.

**Comment by ademclk** [comment]
> > > > > Hi, I can work on this. Could you assign me this issue?
> > > > 
> > > > 
> > > > Done.
> > > 
> > > 
> > > Thanks! Can I submit the solution in Python or Java? I see there's already a C++ solution
> > 
> > 
> > Yes you can submit any language you want. If you've different approach you can submit the cpp version, too.
> 
> Great, I just submitted a python solution with a proposed (more) efficient solution in the description. I would love to implement that later as well, but I can't do that due to time constraints right now.

I made a pull request with your branch and merged to main. Have a nice day.
